fcp-0101.md

```diff
@@ -52,6 +52,7 @@ fxp    | Popular device long recommended by the project.
 dc     | Popular device for CardBus card.
 hme    | Built in interface on many supported sparc64 platforms.
 le     | Emulated by QEMU, alternatives don't yet work for mips64.
+nfe    | nfe is a popular 1GB device in many Ausus motherboards
```

Comment:
Remove this line.  No exception is required as nfe shouldn't have been on the list.

Reply:
Thanks. I'll review/update my patch and push it at noon my time (PDT) today.